nemo/constants.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) NVIDIA CORPORATION. All Rights Reserved.
```

Comment:
these constants look like logging only constants. Should we rename the file to reflect that?

Reply:
I think the idea is that we can put environment variables and other Nemo constants into this file as the project progresses